tekton/src/tasks/fvt-launcher/launchfvt-upgrade-pre.yml.j2

```diff
@@ -77,6 +77,12 @@ spec:
               name: mas-fvt-monitor
               key: MAS_APP_CHANNEL_MONITOR
               optional: false
+        - name: MAS_APP_CHANNEL_OPTIMIZER
```

Comment:
Validate if it is created on ansible-fvt side

Reply:
I confirmed it is created by ansible-fvt here: https://github.ibm.com/maximoappsuite/ansible-fvt/blob/master/ansible/ibm/mas_fvt/roles/fvt_launcher/tasks/fvt-secrets.yml#L202-L220